islandora_defaults.views.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * Provide Views data for integer-weight fields.
+ */
+
+/**
+ * Implements hook_views_data_alter().
+ */
+function islandora_defaults_views_data_alter(&$data) {
+  // For now only support Nodes.
+  $fields = \Drupal::service('entity_field.manager')->getFieldStorageDefinitions('node');
+  foreach ($fields as $field => $field_storage_definition) {
+    if ($field_storage_definition->getType() == 'integer' && strpos($field, "field_") === 0) {
```

Comment:
Does this operate on any integer field?

Reply:
Yes, this can work on any integer field. However, I opted to only use those prefixed with 'field_' because otherwise it would give you the option of changing 'nids' which I thought is 1. something we don't want and 2. probably would throw a bunch of errors if you tried anyway.